oz/core/actions.py

```diff
@@ -107,7 +119,15 @@ def server():
             signal.signal(signal.SIGTERM, functools.partial(_shutdown_handler, http_srv))
             signal.signal(signal.SIGINT, functools.partial(_shutdown_handler, http_srv))
 
-        tornado.ioloop.IOLoop.instance().start()
+        if server_type == "asyncio":
```

Comment:
If it's too difficult (or not really possible?) to find an equivalent for asyncio and twister, your solution is fine. It's not something we need to block on